### PR TITLE
Add ignorable_content field to Repository entity

### PR DESCRIPTION
**Pull request by kersommoura**
Add ignorable_content field to Repository entity.

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -6068,6 +6068,7 @@ def __init__(self, server_config=None, **kwargs):
             ),
             'full_path': entity_fields.StringField(),
             'gpg_key': entity_fields.OneToOneField(ContentCredential),
+            'ignorable_content': entity_fields.ListField(),
```
Comment (rochacbruno) [inline nailgun/entities.py]: Is there a fixed choice list to require here?

Comment (kersommoura) [reply nailgun/entities.py]: I will take a look. Thanks.

Comment (kersommoura) [reply nailgun/entities.py]: > List of content units to ignore while syncing a yum repository. Must be subset of rpm,drpm,srpm,distribution,erratum

Comment (rochacbruno) [reply nailgun/entities.py]: I was thinking to add a validator to ensure the values on that list are on that subset, but looks like the `ListtField` does not support `choices` or any other kind of validator. So nm.

**Pull request merged**